ocaml/xapi/context.ml

```diff
@@ -234,17 +234,19 @@ let parent_of_origin (origin : origin) span_name =
   | _ ->
       None
 
-let start_tracing_helper parent_fn task_name =
+let start_tracing_helper ?(span_attributes = []) parent_fn task_name =
```

Comment:
There are some other places that use `start_tracing_helper`, e.g. line 281. Do we need to update those as well?

Reply:
This needs further discussion on what paths we want to instrument with this.

Reply:
After discussing this with @mg12, I extended to list of new possible attributes and added them where appropriate to `start_tracing_helper`.